ui/app/components/keymgmt/key-edit.js

```diff
@@ -54,16 +54,29 @@ export default class KeymgmtKeyEdit extends Component {
     const { model } = this.args;
     try {
       yield model.save();
+      yield model.reload();
```

Comment:
Curious does the response from the save request return no content but values are being updated on the backend?

Reply:
Correct, PUT and POST both result in a 204 httpstatus.

I was being generous with model reloading (because there are a lot of actions that change the data) but it turns out this one is superfluous :)